angular/wolox-books/src/app/screens/register/register.component.html

```diff
@@ -0,0 +1,27 @@
+<form class="register-form" [formGroup]="registerForm" (ngSubmit)="register(registerForm.value)">
+  <img class="logo" src="../../../assets/wolox_logo.svg" />
```

Comment:
we are going to use an image with the books text in it directly to avoid having to add a text below the image

Reply:
ask me for it